ui/redux/actions/claims.js

```diff
@@ -175,8 +196,11 @@ export function doResolveClaimIds(claimIds: Array<string>) {
         }
       )
     );
+
+    return { ...response, ...resolvedClaims };
```

Comment:
A bit concerned with this change.  
- What's the return value for client that doesn't call await (like in the comments action).  
- Will the change in return affect any existing clients?

Will confirm tomorrow...

Anyway, the Top section is not appearing -- not sure if related to this or somewhere else.

<img width="2310" alt="image" src="https://user-images.githubusercontent.com/64950861/189713990-5c8fdef6-72a5-4797-8492-fed7e5d5ef90.png">